src/Microsoft.FSharp.Compiler/Microsoft.FSharp.Compiler.fsproj

```diff
@@ -34,8 +34,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\fsi\fsiProject\fsi.fsproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\fsc\fscProject\fsc.fsproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\fsi\fsiProject\fsi.fsproj">
+        <AdditionalProperties>Configuration=ReleaseCompressed;CompressAllMetadata=true</AdditionalProperties>
```

Comment:
Wouldn't the property setting `CompressAllMetadata=true` be sufficient?
Or is there another reason for having a separate Configuration I am not seeing?

Reply:
No, fsc and fsi, have the same set of projects as dependencies as appear later on.  The later on set need to be either release or debug, with the standard compressions.  The reason for this is to ensure that the fsi and fsc that we embed in the sdk have the correct (I.e. compressed state, and the fsharp.compiler.service and fsharp.core nuget packages that are also embedded also have the correct, I.e not compressed state.